docs/about/contributing/modules.md

```diff
@@ -0,0 +1,395 @@
+
+## What are Coder modules
+
+Coder modules are Terraform modules that integrate with Coder workspaces to provide specific functionality. They are published to the Coder Registry at [registry.coder.com](https://registry.coder.com) and can be consumed in any Coder template using standard Terraform module syntax.
+
+Examples of modules include:
+
+- **IDE integrations**: [`code-server`](https://registry.coder.com/modules/coder/code-server), [`jetbrains-gateway`](https://registry.coder.com/modules/coder/jetbrains-gateway), [`cursor`](https://registry.coder.com/modules/coder/cursor)
+- **Development tools**: [`git-clone`](https://registry.coder.com/modules/coder/git-clone), [`dotfiles`](https://registry.coder.com/modules/coder/dotfiles), [`devcontainers-cli`](https://registry.coder.com/modules/coder/devcontainers-cli)
+- **Workspace utilities**: [`filebrowser`](https://registry.coder.com/modules/coder/filebrowser), [`coder-login`](https://registry.coder.com/modules/coder/coder-login), [`personalize`](https://registry.coder.com/modules/coder/personalize)
+
+## Prerequisites
+
+Before contributing modules, ensure you have:
+
+- Basic Terraform knowledge
+- [Terraform installed](https://developer.hashicorp.com/terraform/install)
+- [Docker installed](https://docs.docker.com/get-docker/) (for running tests)
+- [Bun installed](https://bun.sh/docs/installation) (for running tests and tooling)
+
+## Setup your development environment
+
+1. **Fork and clone the repository**:
+
+   ```bash
+   git clone https://github.com/your-username/registry.git
+   cd registry
+   ```
+
+2. **Install dependencies**:
+
+   ```bash
+   bun install
+   ```
+
+3. **Understand the structure**:
+
+   ```text
+   registry/[namespace]/
+   ├── modules/         # Your modules
+   ├── .images/         # Namespace avatar and screenshots
+   └── README.md        # Namespace description
+   ```
+
+## Create your first module
+
+### 1. Set up your namespace
+
+If you're a new contributor, create your namespace directory:
+
+```bash
+mkdir -p registry/[your-username]
+mkdir -p registry/[your-username]/.images
+```
+
+Add your namespace avatar by downloading your GitHub avatar and saving it as `avatar.png`:
+
+```bash
+curl -o registry/[your-username]/.images/avatar.png https://github.com/[your-username].png
+```
+
+Create your namespace README at `registry/[your-username]/README.md`:
+
+```markdown
+---
+display_name: "Your Name"
+bio: "Brief description of what you do"
+github: "your-username"
+avatar: "./.images/avatar.png"
+linkedin: "https://www.linkedin.com/in/your-username"
+website: "https://your-website.com"
+status: "community"
+---
+
+# Your Name
+
+Brief description of who you are and what you do.
+```
+
+**Note**: The `linkedin` and `website` fields are optional and can be omitted or left empty if not applicable.
```

Comment:
```suggestion
> [!NOTE]
> The `linkedin` and `website` fields are optional and can be omitted or left empty if not applicable.
```

always prefer GFM-style alerts. @EdwardAngert, can we add a linting rule for these?